.github/workflows/int-test-workflow.yml

```diff
@@ -56,19 +57,8 @@ jobs:
             appframeworks1,
             managerappframeworkc3,
             managerappframeworkm4,
-            managersecret,
-            managersmartstore,
-            managermc,
-            managercrcrud,
-            licensemanager,
             masterappframeworkc3,
             masterappframeworkm4,
-            mastersecret,
-            mastersmartstore,
-            mastermc,
-            mastercrcrud,
-            licensemaster,
-            managerdeletecr,
```

Comment:
is this correct? why did we remove these tests?

Reply:
This was done to get quicker pipeline results for the suites that were concerning to us. This file would be reverted pre-merge